internal/errs/errs.go

```diff
@@ -52,12 +58,21 @@ func (e *WrapperError) Stack() *stacktrace.Stacktrace {
 	return e.stack
 }
 
+func (e *WrapperError) ExitCode() int {
+	return e.code
+}
+
+func (e *WrapperError) SetExitCode(code int) {
+	e.code = code
+}
+
```

Comment:
IMO, this type is doing too much; I suggest adding behavior to error chains using structure rather than an OOP-like "superclass":
```go
type ExitError struct {
	Code int
	Err error
}

func (e *ExitError) Unwrap() error {
	return e.Err
}

func (e *ExitError) ExitCode() int {
	return e.Code
}
```
This can be easily added to error chains that should propagate exit codes:
```go
	return nil, &errs.ExitError{42, err}
```
Then, the interface can be freed up:
```go
type ExitCoder interface {
	ExitCode() int
}
```